hexrd/instrument.py

```diff
@@ -3672,18 +3674,66 @@ def __init__(self, image_series_dict, instrument, plane_data,
             self._iHKLList = active_hkls
             n_rings = len(active_hkls)
 
+        # grab a det key and corresponding imageseries (first will do)
+        # !!! assuming that the imageseries for all panels
+        #     have the same length and omegas
+        det_key = list(eta_mapping.keys())[0]
+        this_det_ims = image_series_dict[det_key]
+
+        # handle omegas
+        # !!! for multi wedge, enforncing monotonicity
+        # !!! wedges also cannot overlap or span more than 360
+        omegas_array = this_det_ims.metadata['omega']  # !!! DEGREES
+        frame_mask = None
+        ome_period = omegas_array[0, 0] + 360.  # !!! be careful
```

Comment:
Is the official way to get the period via the `hexrd.findorientations._process_omegas()` function being called on the imageseries dict? Maybe we can do that?